app/views/conferences/show.html.haml

```diff
@@ -61,6 +61,10 @@
       %section#tickets
         = render 'tickets'
 
+    - if @conference.booths.any? and @conference.splashpage.include_booths
```

Comment:
We only want to show the booths section if there are confirmed booths, don't we?